daprdocs/content/en/getting-started/quickstarts/workflow-quickstart.md

```diff
@@ -254,8 +254,256 @@ The `Activities` directory holds the four workflow activities used by the workfl
 - `ProcessPaymentActivity.cs`
 - `UpdateInventoryActivity.cs`
 
+{{% /codetab %}}
+
+ <!-- Python -->
+{{% codetab %}}
+
+### Step 1: Pre-requisites
+
+For this example, you will need:
+
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started).
+- [Python 3.7+ installed](https://www.python.org/downloads/).
+<!-- IGNORE_LINKS -->
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+<!-- END_IGNORE -->
+
+### Step 2: Set up the environment
+
+Clone the [sample provided in the Quickstarts repo](https://github.com/dapr/quickstarts/tree/master/workflows).
 
+```bash
+git clone https://github.com/dapr/quickstarts.git
+```
+
+In a new terminal window, navigate to the `order-processor` directory:
+
+```bash
+cd workflows/python/sdk/order-processor
+```
+
+### Step 3: Run the order processor app
+
+In the terminal, start the order processor app alongside a Dapr sidecar:
+
+```bash
+pip3 install -r requirements.txt
+dapr run --app-id order-processor --components-path ../../../components/ -- python3 app.py
+```
 
+This starts the `order-processor` app with unique workflow ID and runs the workflow activities. 
+
+Expected output:
+
+```
+== APP == *** Welcome to the Dapr Workflow console app sample!
+== APP == *** Using this app, you can place orders that start workflows.
+== APP == 2023-06-06 09:35:52.892 durabletask-worker INFO: Starting gRPC worker that connects to 127.0.0.1:65406
+== APP == item: InventoryItem(item_name=Paperclip, per_item_cost=5, quantity=100)
+== APP == item: InventoryItem(item_name=Cars, per_item_cost=15000, quantity=100)
+== APP == item: InventoryItem(item_name=Computers, per_item_cost=500, quantity=100)
+== APP == ==========Begin the purchase of item:==========
+== APP == Starting order workflow, purchasing 11 of cars
+== APP == 2023-06-06 09:35:52.945 durabletask-worker INFO: Successfully connected to 127.0.0.1:65406. Waiting for work items...
```

Comment:
We can probably remove a lot of the lines similar to this one. While they are in the output, they are more so internal logging statements that are relevant to how the underlying engine is operating and I don't think it would be super necessary for the user to know.

It the .NET example we only really put the statements relevant to the app inside this section.

Lines like this can probably be removed:
== APP == 2023-06-06 09:35:56.001 durabletask-worker INFO: f4e1926e-3721-478d-be8a-f5bebd1995da: Waiting for 1 task(s) and 0 event(s).

Reply:
Resolved. @RyanLettieri , can you take a look and let me know if it looks better?